about.html

```diff
@@ -423,6 +479,56 @@
   {% endfor %}
   </div>
 
+  {% include hr.html %}
+
+    <!-- Executive Council -->
+    <div class="content content--copy">
+        <h2 class="text-center">Executive Council Members</h2>
+    </div>
+
```

Comment:
I think it would be good to have a short description of what the EC/SSC are here. Something higher level than just the governance.

Reply:
+1; existing description: https://jupyter.org/governance/software_steering_council.html

Reply:
We can pull from and link to http://jupyter.org/governance/executive_council.html and http://jupyter.org/governance/software_steering_council.html

Reply:
+1 to making these links

Reply:
Done in the latest commit: https://deploy-preview-712--jupyter-github-io.netlify.app/about

Reply:
In another commit, I copied the descriptions from the governance docs into the about page to make it more friendly to read.